tldw_Server_API/app/main.py

```
@@ -5692,6 +5693,13 @@ async def api_metrics():
         app.include_router(billing_webhooks_router, prefix=f"{API_V1_PREFIX}", tags=["billing"])
     except _IMPORT_EXCEPTIONS as _billing_webhooks_min_err:
         logger.debug(f"Skipping billing webhooks router in minimal test app: {_billing_webhooks_min_err}")
+    # Consent management endpoints
+    try:
+        from tldw_Server_API.app.api.v1.endpoints.consent import router as consent_router
+
+        app.include_router(consent_router, prefix=f"{API_V1_PREFIX}", tags=["consent"])
+    except _IMPORT_EXCEPTIONS as _consent_min_err:
+        logger.debug(f"Skipping consent router in minimal test app: {_consent_min_err}")
```

Ruff (0.15.6)

[warning] Logging statement uses f-string

(G004)